README.md

```diff
@@ -10,13 +10,12 @@
 
 This repository contains a set of Dockerfiles which can be used to develop, build, and run Covalent. These images will not change very often, as they are used to set up build environments but do not include the actual dependencies of Covalent. The following environments are included:
 
-- All combinations of Debian 10, 11 and Python 3.8, 3.9, 3.10
-- CentOS 7 and Python 3.8
+- All combinations of Debian 11, 12, and Python 3.9, 3.10, 3.11
 
-To build a single one of these images locally, e.g., Debian 10 with Python 3.8, run the following:
+To build a single one of these images locally, e.g., Debian 11 with Python 3.10, run the following:
```

Comment:
Should this say `Debian 12` given the command below?

Reply:
Nice catch, thanks.